Sources/Vapor/Routing/RoutesBuilder+Group.swift

```diff
@@ -13,9 +13,24 @@ extension RoutesBuilder {
     ///     - path: Group path components separated by commas.
     /// - returns: Newly created `Router` wrapped in the path.
     public func grouped(_ path: PathComponent...) -> RoutesBuilder {
+        return self.grouped(path)
+    }
+
+    /// Creates a new `Router` that will automatically prepend the supplied path components.
+    ///
+    ///     let users = router.grouped("user")
```

Comment:
This example should be updated to reflect the array usage IMO (but feel free to ignore. Is a minor issue :)
```suggestion
    ///     let users = router.grouped(["user"])
```